server/services/visitService.ts

```diff
@@ -1,8 +1,12 @@
+import { v4 as uuidv4 } from 'uuid'
 import logger from '../../logger'
 import { BookingJourney } from '../@types/bapv'
 import { RestClientBuilder, OrchestrationApiClient, HmppsAuthClient } from '../data'
 import { ApplicationDto, OrchestrationVisitDto, VisitDto } from '../data/orchestrationApiTypes'
 
+export interface VisitDetails extends OrchestrationVisitDto {
```

Comment:
New `VisitDetails` type which has additional `visitDisplayId` to hold UUID generated to reference it.